foundation/sponsors.dd

```diff
@@ -77,6 +77,11 @@ $(UL
     $(LI Paul)
     $(LI Alex Semenikhine)
     $(LI Sebastian Wilzbach)
+    $(LI Ross Lonstein)
+    $(LI Matthew Strawbridge)
+    $(LI Andrii Bazylevych)
+    $(LI Jeffrey Buhr)
+    $(LI Aaron Disibio)
```

Comment:
Glad to see the new names. The existing list needs to be reordered to match the gist I sent you, and the new names inserted based on their total donation. I'll email the gist link in case you don't have it anymore.